cmd/daprd/components/binding_kitex.go

```diff
@@ -0,0 +1,26 @@
+//go:build !(linux && arm)
+// +build !linux !arm
```

Comment:
note: Using the Go build tag to skip testing is not compiling on the Linux ARM platform.